### ISSUE TITLE
Add the following health-related fields to this page with corresponding back-end mods

### ISSUE THREAD

**Issue by wclenhardt** [myhpom/MyHPOM#72]
Health Pieces That Matter:
Vitals (height, weight, BP, blood type) 
Immunizations
Allergies
Medications
Current medical conditions
Organ Donor
Video to others? 

There is a functionality request to allow the user to export a 'snapshot' of this information; probably in the form of a pdf.  I will flag this as a separate issue for now. But I expect we'll need to discuss further.  If so, just let me know.

**Comment by cbcunc** [comment]
These are not fields. They look like whole new document types. Are they free form or defined metadata? What is "this page?" Please reassign to me when you have an idea of what you want. Not currently in the Release 1.0 Milestone for March 28. I assume this relates to #71 and #73?

**Comment by cbcunc** [comment]
I assume this relates to #71 and #73?

**Comment by wclenhardt** [comment]
@cbcunc Yes, a more fully specified description of this will have to be pulled together so we can link these issues to a future milestone.  Thx.
